modules/util/source-formatter/src/main/resources/dependencies/replacements.json

```diff
@@ -3274,6 +3274,19 @@
 		"issueKey": "LPD-36261",
 		"to": "fetchColumn"
 	},
+	{
+		"from": "_groupPermission.contains",
+		"issueKey": "LPD-36287",
+		"newImports": [
+			"com.liferay.portal.kernel.service.permission.GroupPermissionUtil"
+		],
+		"to": "GroupPermissionUtil.contains",
+		"validExtensions": [
+			"java",
+			"jsp",
+			"jspf"
```

Comment:
I don't think we will be able to validate the type using classNames for other types besides java, consult the confluence page.